views/Blog.js

```diff
@@ -1,156 +1,29 @@
-							</FacebookShareButton>
-							<TwitterShareButton url='#'>
-								<TwitterIcon size={32} round={true} />
-							</TwitterShareButton>
-						</div>
-						<h3 className={styles.Title}>Lorem ipsum</h3>
-						<Link href='/post'>
-							<a aria-label='Next'>
-								<ArrowRight className={styles.ArrowRight} />
-							</a>
-						</Link>
-					</div>
-					<div className={styles.PreviewWrap}>
-						<div className={styles.ImageWrap}>
-							<Image
-								src='https://picsum.photos/200'
-								layout='fill'
-								alt='image'
-							/>
-						</div>
-						<div className={styles.PostAuthorDataWrap}>
-							<div className={styles.TextWrap}>
-								<p className={styles.Text}>Erick Askim</p>
-								<p className={styles.Text}>Aug 12,2021</p>
-							</div>
-							<FacebookShareButton url='#'>
-								<FacebookIcon size={32} round={true} />
-							</FacebookShareButton>
-							<TwitterShareButton url='#'>
-								<TwitterIcon size={32} round={true} />
-							</TwitterShareButton>
-						</div>
-						<h3 className={styles.Title}>Lorem ipsum</h3>
-						<Link href='/post'>
-							<a aria-label='Next'>
-								<ArrowRight className={styles.ArrowRight} />
-							</a>
-						</Link>
-					</div>
-					<div className={styles.PreviewWrap}>
-						<div className={styles.ImageWrap}>
-							<Image
-								src='https://picsum.photos/200'
-								layout='fill'
-								alt='image'
-							/>
-						</div>
-						<div className={styles.PostAuthorDataWrap}>
-							<div className={styles.TextWrap}>
-								<p className={styles.Text}>Erick Askim</p>
-								<p className={styles.Text}>Aug 12,2021</p>
-							</div>
-							<FacebookShareButton url='#'>
-								<FacebookIcon size={32} round={true} />
-							</FacebookShareButton>
-							<TwitterShareButton url='#'>
-								<TwitterIcon size={32} round={true} />
-							</TwitterShareButton>
-						</div>
-						<h3 className={styles.Title}>Lorem ipsum</h3>
-						<Link href='/post'>
-							<a aria-label='Next'>
-								<ArrowRight className={styles.ArrowRight} />
-							</a>
-						</Link>
-					</div>
-				</div>
-				<div className={styles.PaginationWrap}>
-					<ul className={styles.PaginationUl}>
-						<li className={styles.PaginationLi}>
-							<Link href='/post'>
-								<a className={styles.Link}>1</a>
-							</Link>
-						</li>
-						<li className={styles.PaginationLi}>
-							<Link href='/post'>
-								<a className={styles.Link}>2</a>
-							</Link>
-						</li>
-						<li className={styles.PaginationLi}>
-							<Link href='/post'>
-								<a className={styles.Link}>3</a>
-							</Link>
-						</li>
+				<Head>
+					<script src='https://identity.netlify.com/v1/netlify-identity-widget.js'></script>
+				</Head>
+				<article>
+					<h1>{title}</h1>
+					<HomeContent />
+					<ul>
+						{posts.map((post, k) => (
+							<li key={k}>
+								<h2>{post.name}</h2>
+								<p>{post.description}</p>
+							</li>
+						))}
```

Comment:
It's not going to work this way, but you'll know why after you use the cms to make some posts.

Reply:
You can just resolve this convo, no actions needed.